datadogfirehosenozzle/datadog_firehose_nozzle.go

```diff
@@ -231,18 +264,20 @@ func (d *DatadogFirehoseNozzle) PostMetrics() {
 	totalMessagesReceived := d.totalMessagesReceived
 	d.mapLock.Unlock()
 
-	// Add internal metrics
-	k, v := d.client.MakeInternalMetric("totalMessagesReceived", totalMessagesReceived)
-	metricsMap[k] = v
-	k, v = d.client.MakeInternalMetric("totalMetricsSent", d.totalMetricsSent)
-	metricsMap[k] = v
-	k, v = d.client.MakeInternalMetric("slowConsumerAlert", atomic.LoadUint64(&d.slowConsumerAlert))
-	metricsMap[k] = v
+	timestamp := time.Now().Unix()
```

Comment:
this might happen quickly enough that the timestamps are the same, though grabbing them only once is likely the best optimization you can do here.

Reply:
I want to make sure the data is the exact same